snap/bin/stop-edgex.sh

```diff
@@ -95,12 +95,30 @@ if [ $SUPPORT_LOGGING = "y" ] ; then
     fi
 fi
 
+if [ $SECURITY ]; then
+    # stop kong - note this might fail, so we just force it to always succeed with "|| true"
+    $SNAP/bin/kong-wrapper.sh stop -p $SNAP_DATA/kong || true
+
+    # killing nginx is tricky, as we need to make sure we don't inadvertantly kill any other nginx
```

Comment:
Isn't nginx part of kong? I'm surprised we need to add extra code to stop it.  Does the docker kong image work the same way on shutdown?

Reply:
Yes, nginx is part of kong. We need extra code here because the above kong stop command doesn't always work.
No, the docker kong image doesn't work the same way on shutting down. The dockerfile doesn't actually specify any shutdown commands. I assume this is because stopping the container will end all child processes forcefully.